cookbook/integrations/flytekit_plugins/mlflow_example/README.rst

```diff
@@ -0,0 +1,42 @@
+.. mlflow:
+
+MLflow
+======
+
+.. tags:: Integration, Data, Metrics, Intermediate
+
+The MLflow Tracking component is an API and UI for logging parameters,
+code versions, metrics, and output files when running your machine learning code and for later visualizing the results
+
+`MLflow <https://mlflow.org/>`__ is used by Flyte to automatically log the task's metrics and parameters to either Flyte Deck or MLflow server.
+
+.. figure:: https://user-images.githubusercontent.com/37936015/200655711-8eb54757-cc08-4591-8f44-787cc4b0af66.png
+  :alt: Model Metrics
+  :class: with-shadow
+
+.. figure:: https://user-images.githubusercontent.com/37936015/200655752-fddfd0cd-26f2-4ccf-914a-08044c7c4dae.png
+  :alt: Model Parameters
+  :class: with-shadow
```

Comment:
I think we can remove this from the integration entrypoint page, since it also appears on the example page